### PR TITLE
created angular tasks and committed to the feature branch 

### PR DESCRIPTION
**Pull request by amshekar**
@maheshbhuvanagiri 
Please review the tasks and and provide your review comments as much as possible.
So i can learn more by implementing review comments.

### CODE REVIEW

File: home-work-4/app-cat-clicker/premiumcat/premiumcat.service.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+﻿(function (module) {
+    function premiumCatService($http, $q, $location) {
```
Comment (maheshbhuvanagiri) [inline home-work-4/app-cat-clicker/premiumcat/premiumcat.service.js:2]: none of the services($http,$q,$location) are being used inside service function. use $q to create the promises and resolve the data.

File: home-work-4/app-cat-clicker/premiumcat/premiumcat.service.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            name: 'Asian',
+            url: 'http://lh3.ggpht.com/nlI91wYNCrjjNy5f-S3CmVehIBM4cprx-JFWOztLk7vFlhYuFR6YnxcT446AvxYg4Ab7M1Fy0twaOCWYcUk=s0#w=640&amp;h=426',
+
+        }, {
+            name: 'Asian Semi-longhair',
+            url: 'http://lh3.ggpht.com/kixazxoJ2ufl3ACj2I85Xsy-Rfog97BM75ZiLaX02KgeYramAEqlEHqPC3rKqdQj4C1VFnXXryadFs1J9A=s0#w=640&h=496'
+
+        }];
+
+        return service;
```
Comment (maheshbhuvanagiri) [inline home-work-4/app-cat-clicker/premiumcat/premiumcat.service.js]: return statement should be the last statement of the function, though you have GetCats() below this return statement which gets hoisted, its better to put the return as the last statement

File: home-work-4/app-cat-clicker/singlecat/singlecat.controller.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        var vm = this;
+        vm.count = 0;
+        vm.catClick = catClick;
+
+        function catClick() {
+            vm.count++;
+        }
+
+    }
+    SingleCatController.$inject = [];
```
Comment (maheshbhuvanagiri) [inline home-work-4/app-cat-clicker/singlecat/singlecat.controller.js]: not required, use only when u need to inject any dependencies

File: home-work-4/app-cat-clicker/twocats/twocats.controller.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+            vm.cat1Clik++;
+        }
+
+        function clickCount2() {
+            vm.cat2Clik++;
+        }
+
+    }
+
+    TwoCatsController.$inject = [];
```
Comment (maheshbhuvanagiri) [inline home-work-4/app-cat-clicker/twocats/twocats.controller.js]: This statement is not needed, as ur not injecting any dependencies

File: home-work-4/app-cat-clicker/index.html [hunk 5]
```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.6/angular.js"></script>
+    <link rel="stylesheet" href="css/bootstrap.css">
+    <link rel="stylesheet" href="css/customstyles.css">
+    <script src="app.module.js"></script>
```
Comment (maheshbhuvanagiri) [inline home-work-4/app-cat-clicker/index.html]: load css files before the js files

**Comment by amshekar** [comment]
@maheshbhuvanagiri  Implemented all review comments. Please review

**Pull request merged**